options/locale/locale_en-US.ini

```
@@ -1417,7 +1417,7 @@ issues.due_date_remove = "removed the due date %s %s"
 issues.due_date_overdue = "Overdue"
 issues.due_date_invalid = "The due date is invalid or out of range. Please use the format 'yyyy-mm-dd'."
 issues.dependency.title = Dependencies
-issues.dependency.issue_no_dependencies = This issue currently doesn't have any dependencies.
+issues.dependency.issue_no_dependencies = No dependencies set.
 issues.dependency.pr_no_dependencies = This pull request currently doesn't have any dependencies.
```

What I meant is simply:
```suggestion
issues.dependency.pr_no_dependencies = No dependencies set.
```

---

I'm not sure what you mean.
Changing lines inside `options/locale/locale_en-US.ini` automatically notifies translators to provide a new translation for this string as soon as this PR is merged (-> Crowdin).

---

Oh, i see now, there was another spot in the same file that needed the same change! Sorry about the confusion. 
Thanks for the clarification about the translators being notified 👍